src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.setRadius(radius);
+        this.setColor(color);
+    }
```

Comment:
```suggestion
    public Circle(int radius, String color) {
        this.radius = radius;
        setColor(color);
    }
```

same for other figures